### DISCUSSION TITLE
Tuya Blinds Precisely Control

### DISCUSSION THREAD

**Discussion by bincvburen** [milo526/homebridge-tuya-web#291] [Ideas]
Hi! Thanks for your great plug-in! Is it possible to have a percent based controller instead of just opens & close in the home app. I have Tuya smart blinds controller and I can control them via the Tuya app precisely. Is it possible to bring that to the home app too?

**Comment by milo526** [comment]
This is discussed in issue https://github.com/milo526/homebridge-tuya-web/issues/148
